src/clr-angular/image/_icons.clarity.scss

```diff
@@ -76,13 +76,6 @@ $svg_data: 'data:image/svg+xml;charset=utf8';
   @return url('#{$svg_data},#{$right-caret-icon}');
 }
 
-@function generateFilterIcon($fillColor: $clr-color-action-600) {
-  $filter-off-icon: '%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20viewBox%3D%220%200%2016%2016%22%3E%3Cdefs%3E%3Cstyle%3E.cls-1%7Bfill%3A%23' +
-    str-slice($fillColor + '', 2) +
-    '%3Bfill-rule%3Aevenodd%3B%7D%3C%2Fstyle%3E%3C%2Fdefs%3E%3Cpolygon%20class%3D%22cls-1%22%20points%3D%227%209.32%203%205.38%203%205%2013%205%2013%205.38%209%209.32%209%2012.21%207%2013.29%207%209.32%22%2F%3E%3C%2Fsvg%3E';
-  @return url('#{$svg_data},#{$filter-off-icon}');
-}
-
 @function generateCircleIcon($fillColor: $clr-color-action-600) {
```

Comment:
I believe you have also removed the only usage of `generateCircleIcon` in our library and can safely remove this function as well...

Reply:
Right, I can't find any usages for it. I'll remove and push a second commit.

Reply:
😂 That was fast!

Great work. I'm happy to see these start going away.